src/utils/RemindersContext.tsx

```diff
@@ -44,19 +48,29 @@ const RemindersContext = createContext<RemindersContextType | undefined>(
 export const RemindersProvider: React.FC<{ children: React.ReactNode }> = ({
   children,
 }) => {
-  const [lists, setLists] = useState<List[]>([]);
+  const [lists, setLists] = useState<List[]>([
+    { id: 'Today', name: 'Today', tasks: [] },
```

Comment:
I'd recommend making these `id` values an enum, but I also think that having them as lists themselves is the wrong data model.